generators/entity-client/templates/react/src/main/webapp/app/entities/entity-dialog.tsx.ejs

```diff
@@ -44,14 +59,19 @@ _%>
 import { AvForm, AvGroup, AvInput<% if (isAvField) { %>, AvField <% } %> } from 'availity-reactstrap-validation';
 // tslint:disable-next-line:no-unused-variable
 import { Translate, translate, ICrudGetAction, ICrudGetAllAction, ICrudPutAction } from 'react-jhipster';
-import { FaBan, FaFloppyO } from 'react-icons/lib/fa';
+import { FaBan, <% if(hasBlob) { %>FaTimesCircle, <% } %>FaFloppyO } from 'react-icons/lib/fa';
+<%_ if (hasBlob) { _%>
+import { setFileData, clearInputImage,<% if(hasAnyBlobOrImage) { %> openFile,<% } %> byteSize } from 'app/shared/util/data-utils';
```

Comment:
where is `clearInputImage` used?

Reply:
It's not used anymore, I'll remove it from both `data-utils` and `entity-dialog`.